onnx/test/training_tool_test.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import onnx
+import numpy as np
+from onnx.tools import update_model_dims
+from onnx import helper, numpy_helper, TensorProto
+
+
+class TestTrainingTool(unittest.TestCase):
+    def test_training_info_proto(self):  # type: () -> None
+        # Inference graph.
+        A_shape = [2, 2]
+        A_name = 'A'
+        A = np.array(A_shape).astype(np.float)
+        A_initializer = numpy_helper.from_array(A, name=A_name)
+        A_value_info = helper.make_tensor_value_info(
+            A_name, TensorProto.FLOAT, A_shape)
+
+        B_shape = [2, 2]
+        B_name = 'B'
+        B = np.array(B_shape).astype(np.float)
```

Comment:
This will create an np array [2, 2] with shape [2] whereas on line 27 when creating B_value_info the shape is set as [2,2]. What am I missing here?

Reply:
My bad. I should create a random function here. 

My mistake here is a good example to support the need of having a runtime test (either ORT or a small runtime) in ONNX repo.